test/test_uninstall_and_reconcile_endpoint.py

```
@@ -0,0 +1,311 @@
+from flask import Flask
+
+
+def _make_client():
+    """Minimal Flask app + mocked deps that exercises the api_v3 blueprint."""
+    from web_interface.blueprints import api_v3 as api_v3_module
+    from web_interface.blueprints.api_v3 import api_v3
+
+    # Mocks for all the bits the reconcile / uninstall endpoints touch.
+    api_v3.config_manager = MagicMock()
+    api_v3.config_manager.get_raw_file_content.return_value = {}
+    api_v3.config_manager.secrets_path = "/tmp/nonexistent_secrets.json"
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/nonexistent_secrets.json"

(S108)